<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cobrar_</class>
 <widget class="QWidget" name="cobrar_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#0c0c0c;
color:white;
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>391</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1D1D1D;
border-radius: 17px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="CONTENEDOR">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>GENERAR TICKET</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_agregar">
             <property name="text">
              <string>Agregar a ticket</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_cancelar">
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0" rowspan="3">
        <widget class="QTableView" name="Table_platillos_Activos"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>331</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1D1D1D;
border-radius: 15px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>258</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="table_ticket"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="LB_PAGAR">
         <property name="text">
          <string>TOTAL A PAGAR:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="COBRAR">
         <property name="text">
          <string>138.87</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cobrar">
       <property name="styleSheet">
        <string notr="true">background-color:#c3c3c3;

color:black;</string>
       </property>
       <property name="text">
        <string>COBRAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
